cocos/core/director.ts

```diff
@@ -740,51 +637,6 @@ export class Director extends EventTarget {
         return this._scene;
     }
 
-    /**
-     * @en Returns the FPS value. Please use [[Game.setFrameRate]] to control animation interval.
-     * @zh 获取单位帧执行时间。请使用 [[Game.setFrameRate]] 来控制游戏帧率。
-     * @deprecated since v2.0.
-     */
-    public getAnimationInterval () {
-        return 1000 / legacyCC.game.getFrameRate();
-    }
-
-    /**
-     * @en Sets animation interval, this doesn't control the main loop.<br>
-     * To control the game's frame rate overall, please use `game.setFrameRate`
-     * @zh 设置动画间隔，这不控制主循环。<br>
-     * 要控制游戏的帧速率，请使用 `game.setFrameRate`
-     * @deprecated since v2.0
-     * @param value - The animation interval desired.
-     */
-    public setAnimationInterval (value: number) {
-        legacyCC.game.setFrameRate(Math.round(1000 / value));
-    }
-
-    /**
-     * @en Returns the delta time since last frame.
-     * @zh 获取上一帧的增量时间。
-     */
-    public getDeltaTime () {
-        return this._deltaTime;
-    }
-
-    /**
-     * @en Returns the total passed time since game start, unit: ms
-     * @zh 获取从游戏开始到现在总共经过的时间，单位为 ms
-     */
-    public getTotalTime () {
-        return performance.now() - this._startTime;
-    }
-
-    /**
-     * @en Returns the current time.
-     * @zh 获取当前帧的时间。
-     */
-    public getCurrentTime () {
-        return this._lastUpdate;
-    }
-
```

Comment:
should we use 
```
/**
     * @en Returns the delta time since last frame.
     * @zh 获取上一帧的增量时间。
     * @depreacated since v3.3, please use game.getDeltaTime() instead
     */
    public getDeltaTime () { }

    /**
     * @en Returns the total passed time since game start, unit: ms
     * @zh 获取从游戏开始到现在总共经过的时间，单位为 ms
     * @depreacated since v3.3, please use game.getTotalTime() instead
     */
    public getTotalTime () { }

    /**
     * @en Returns the current time.
     * @zh 获取当前帧的时间。
     * @depreacated since v3.3, please use game.getCurrentTime() instead
     */
    public getCurrentTime () { }
```

so that developers can see the deprecation info when they are coding in VSCode

Reply:
That's a valid suggestion, thanks